lib/iris/util.py

```diff
@@ -1706,7 +1706,8 @@ def _meshgrid(*xi, **kwargs):
 def find_discontiguities(cube, rel_tol=1e-5, abs_tol=1e-8):
     """
     Searches coord for discontiguities in the bounds array, returned as a
-    boolean array (True where discontiguities are present).
+    boolean array (True for all cells which are discontiguous with the cell
```

Comment:
(sorry, not your fault but ...)
`Searches coord` here is actually wrong
It actually searches the X and Y coords of the passed cube.

We should also be clearly stating that the cube **_must_** have 2d X and Y coordinates.

Reply:
addressed by https://github.com/SciTools/iris/pull/4975/commits/d85ed08c07fe8dd8dce90e610d563f7078920c9d